src/web/routes/application/card-address/card-address.test.js

```diff
@@ -0,0 +1,38 @@
+const test = require('tape')
+const { assocPath } = require('ramda')
+const { contentSummary } = require('./card-address')
+
+const req = {
+  t: string => string,
+  session: {
+    claim: {
+      addressLine1: 'Flat b',
```

Comment:
Minor - we have these address details stored and reused in `constants.js` already but this is in the acceptance steps package, can/should we reuse this here? I'm not sure what you'd normally do here...